src/chainparams.cpp

```diff
@@ -962,6 +966,8 @@ class CRegTestParams : public CChainParams {
         pchMessageStart[3] = 0xdc;
         nDefaultPort = 19899;
         nPruneAfterHeight = 1000;
+        m_assumed_blockchain_size = 0;
+        m_assumed_chain_state_size = 0;
```

Comment:
13216: add the same for devnet

Reply:
Not sure what was meant by this

Reply:
I mean similar changes should be done here https://github.com/dashpay/dash/blob/d21dc246974c59bb67dc1dde44ac917c9694536e/src/chainparams.cpp#L813